ambari-server/src/main/java/org/apache/ambari/server/controller/internal/BlueprintConfigurationProcessor.java

```diff
@@ -499,6 +499,27 @@ private void doNameNodeHAUpdateOnClusterCreation(Configuration clusterConfig,
         if (parsedNameServices.length > 1) {
           Set<String> activeNameNodeHostnames = new HashSet<>();
           Set<String> standbyNameNodeHostnames = new HashSet<>();
+          String tagsyncNameserviceMappingProperty   = "ranger.tagsync.atlas.hdfs.instance." + getClusterName() + ".%s.ranger.service";
```

Comment:
Please store `clusterName` in a local variable and use it instead of multiple `getClusterName()` calls.